docs/en/observability/monitor-infra/analyze-hosts.asciidoc

```diff
@@ -145,3 +145,55 @@ From the **Actions** menu, you can choose to:
 
 [role="screenshot"]
 image::images/hosts-view-alerts.png[Screenshot showing Alerts view]
+
+[discrete]
+[[analyze-hosts-why-dashed-lines]]
+== Why am I seeing dashed lines in charts?
+
+There are a few reasons why you may see dashed lines in your charts.
+
+* <<dashed-interval>>
+* <<dashed-missing>>
+* <<dashed-interval-and-missing>>
+
+[discrete]
+[[dashed-interval]]
+==== The chart interval is too short
+
+In this example, the data emission rate is lower than the Lens chart interval.
+A dashed line connects the known data points to make it easier to visualize trends in the data.
+
+[role="screenshot"]
+image::images/hosts-dashed.png[Screenshot showing dashed chart]
+
+The chart interval is automatically set depending on the selected time duration.
+To fix this problem, change the selected time range at the top of the page.
+
+TIP: Want to dig in further while maintaining the selected time duration?
+Hover over the chart you're interested in and select **Options** -> **Open in Lens**.
+Once in Lens, you can adjust the chart interval temporarily. Note that this change is not persisted in the **Hosts** view.
+
+[discrete]
+[[dashed-missing]]
+==== Data is missing
+
+A solid line indicates that the chart interval is set appropriately for the data transmission rate.
+In this example, a solid line turns into a dashed line--indicating missing data.
+You may want to investigate this time period to determine if there is an outage or issue.
+
+[role="screenshot"]
+image::images/hosts-missing-data.png[Screenshot showing missing data]
+
+[discrete]
+[[dashed-interval-and-missing]]
+==== The chart interval is too short and data is missing
+
+In the screenshot below, the data emission rate is lower than the Lens chart interval **and** there is missing data.
```

Comment:
> the data emission rate is lower

Would it make sense to explain that this happens because of the [`period` interval](https://www.elastic.co/guide/en/beats/metricbeat/current/configuration-metricbeat.html#metricset-period) configured in metricbeat or integration package?